docker-compose.yml

```diff
@@ -18,9 +18,9 @@ services:
     ports:
         - "8081:8081"
     environment:
-        - "NETSBLOX_CLOUD=http://cloud:8080"
+        - "NETSBLOX_CLOUD=http://cloud:7777"
         - "NETSBLOX_CLOUD_ID=NetsBloxServices"
-        - "NETSBLOX_CLOUD_SECRET=PUT_SECRET_HERE"
+        - "NETSBLOX_CLOUD_SECRET=PUT_SECRET_HERE"  # TODO: Add the actual secret here!
         - "PORT=8081"
         - "MONGO_URI=mongodb://mongo:27017"
 # Uncomment the following line to use the local services code in the container (for development)
```

Comment:
Slight nit-pick I didn't notice when we made this originally, technically it's not "the following line". Might be worth being more explicit just in case someone isn't familiar with Docker Compose syntax and takes it literally.